cfDNApipe/cfDNA_utils.py

```diff
@@ -1402,11 +1594,19 @@ def DECONVO(ref, mix, delcol_factor=10, iter_num=10, confidence=0.75, w_thresh=1
                 bestMarker = np.zeros([np.size(selectReference, 0), 1])
                 bestMarker[:, 0] = selectMixture
                 bestNumber = selectedNumber
-        t = RobustSVR(
-            bestReference, bestMarker, iter_num=iter_num, confidence=confidence, w_thresh=w_thresh, unknown=unknown
-        )
-        bestNumberHistory.append(bestNumber)
-        proportionDeconvolution[:, i] = t[:, 0]
+        if (isinstance(bestMarker, np.ndarray) and bestMarker.size) or (
+            isinstance(bestMarker, (list, tuple)) and bestMarker
+        ):
```

Comment:
What handles if the if-clause condition is not met? Is an error thrown or does proportionDeconvolution still have some value that is returned?

Reply:
Eventually we get zero proportion values for all tissues, since the list is empty

Reply:
Check out line 1554